index.js

```diff
@@ -7,7 +7,7 @@ var oneArgumentFunctionRegex = /^\w+/;
 
 function reflector(fn) {
   var src;
-  var body = src = fn.toString();
+  var body = src = Function.prototype.toString.call(fn);

```

Comment:
any reason why call toString like this?

Reply:
it work that if toString has been override

Reply:
alright thanks